.travis.yml

```diff
@@ -1,3 +1,4 @@
+# (C) Copyright IBM Corp. 2019, 2020
 language: bash
```

Comment:
I don't think that we have to copyright anything that isn't executed in the browser.  If we are, we probably need the copyright AND the license.  So I'd suggest we only update the dates on any of the files that already have a copyright statement and add the license and copyright to any .js file that doesn't already have it.  The one I still have a question about is the .svg file since it is the IBM logo and we probably want to protect it, I guess, and not license it?  I dunno, we'll have to ask legal about that.

If you could make those changes (and remove the .sh from the repo since it is IBM specific and hopefully we'll get other contributors in the future), I'd appreciate it.  Thanks!

Reply:
I also need to follow up and see if the page source needs copyright, license information if it is viewed in a browser.  I don't think it does because www.ibm.com and www.google.com don't.  But for some reason I thought that it did since technically the code is being distributed to the browser.